test/pages/SearchResultsPage.ts

```diff
@@ -0,0 +1,26 @@
+import { Locator, Page } from "@playwright/test";
+import { BASE_URL } from "../helper-config";
+
+export class SearchResultsPage {
+    page: Page;
+    searchBooksHeading: Locator;
+    bookNameLocator: Locator;
+    authorNameLocator: Locator;
+    static readonly SEARCH_RESULTS_URL: string = `${BASE_URL}/search?title=Robinson&author=Daniel+Defoe`
+
+    constructor({page}: {page: Page}) {
+        this.searchBooksHeading = page.locator('#test-body-mobile #contentHead h1');
+        this.bookNameLocator = page.locator('#searchResults .booktitle a');
+        this.authorNameLocator = page.locator('#searchResults .bookauthor a');
+    }
+
+    async navigateToSearchResults() {
+      this.page.goto(SearchResultsPage.SEARCH_RESULTS_URL);
+    }
```

Comment:
```suggestion
    async navigateToSearchResults() {
      await this.page.goto(SearchResultsPage.SEARCH_RESULTS_URL);
    }
```
Do we need not to wait to go to the URL?